Gruntfile.js

```diff
@@ -267,6 +285,7 @@ module.exports = function( grunt ) {
 	grunt.registerTask( 'readme',     [ 'wp_readme_to_markdown' ] );
 	grunt.registerTask( 'update-pot', [ 'makepot' ] );
 	grunt.registerTask( 'update-mo',  [ 'potomo' ] );
-	grunt.registerTask( 'version',    [ 'replace', 'readme' ] );
+	grunt.registerTask( 'version',    [ 'replace:php', 'replace:readme', 'readme' ] );
+	grunt.registerTask( 'update-fa',  [ 'replace:font_awesome' ] );
```

Comment:
@EvanHerman While a neat idea I think it's mostly just more code to maintain for a feature that changes 2 lines.